HousingSearchApi/V1/Boundary/Requests/GetAllAssetListRequest.cs

```diff
@@ -28,6 +28,9 @@ public class GetAllAssetListRequest : GetAssetListRequest
         [FromQuery(Name = "stepFree")]
         public string StepFree { get; set; }
 
+        [FromQuery(Name = "isApproved")]
```

Comment:
I think this makes it look the approval is against the asset entity whereas the filter query used is against `assetContract.isApproved` property.

Reply:
changed to `contractIsApproved` to make intent clearer